gulpTasks/filetypes.js

```diff
@@ -126,11 +126,12 @@ function generateStrings(json) {
   // helpers do the same, and string lookups are case sensitive. I do that because
```

Comment:
Since the code is open source and anyone can potentially see those comments, I would avoid using the first person singular ("I"). It looks a bit odd to me :)